tests/Microsoft.Azure.Batch.SoftwareEntitlement.Tests/NodeEntitlementsBuilderTests.cs

```diff
@@ -271,5 +272,26 @@ public void WhenInvalidAddress_ReturnsErrorForAddress()
                 entitlement.Errors.Should().Contain(e => e.Contains("address"));
             }
         }
+
+        public class IssuerProperty : NodeEntitlementsBuilderTests
+        {
+            private readonly string _issuer = "https://account.region.batch.azure.test";
+
+            [Fact]
+            public void WhenMissing_BuildReturnsValue()
+            {
+                _commandLine.Issuer = string.Empty;
+                var result = NodeEntitlementsBuilder.Build(_commandLine);
+                result.HasValue.Should().BeTrue();
```

Comment:
I'm not quite sure what is being asserted here (too much magic in Fluent Assertions perhaps).  Are you asserting that the result is merely non-null (e.g. empty string) or are you asserting that a _nonempty value_ is filled in by the Build function?  (The name of the test doesn't really elucidate either.)